rendezvous/README.md

```diff
@@ -184,8 +191,8 @@ The protocol, as described so far, is susceptible to spam attacks from
 adversarial actors who generate a large number of peer identities and
 register under a namespace of interest (e.g., the relay namespace).
 
-It is TBD how exactly the protocol will mitigate such attacks.
-See https://github.com/libp2p/specs/issues/341 for a discussion on this
+It is TBD how exactly such attacks will be mitigated.
+See <https://github.com/libp2p/specs/issues/341> for a discussion on this
```

Comment:
In my eyes the previous wording made it clear, that it is the protocol that _should_ mitigate against the attacks.